CreatubblesAPIClientTests/Spec/Content/ContentResponseHandlerSpec.swift

```diff
@@ -65,6 +65,32 @@ class ContentResponseHandlerSpec: QuickSpec
                     }
                 }
             }
+            
+            it("Should return Connected contents after login")
+            {
```

Comment:
Even though this test passes, returned entries are always an empty array. Is that working as intended?